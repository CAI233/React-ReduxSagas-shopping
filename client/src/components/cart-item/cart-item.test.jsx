import { shallow } from 'enzyme';
import React from 'react';
import CartItem from './cart-item.component';

// 没有什么好测试的，只是显示个参数
describe( "购物车单个产品显示测试",()=>{
    let wrapper, mockProps;
    beforeEach( ()=>{
        mockProps = {
            item: {
                imageUrl: '1.jpg',
                name: 'test',
                price: 10,
                quantity: 1,
            }
        }
        wrapper = shallow(<CartItem { ...mockProps } />);
    } );

    it(" CartItem Snapshot ",()=>{
        expect( wrapper ).toMatchSnapshot();
    });

} );