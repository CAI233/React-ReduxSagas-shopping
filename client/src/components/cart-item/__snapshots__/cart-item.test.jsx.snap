// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`购物车单个产品显示测试  CartItem Snapshot  1`] = `ShallowWrapper {}`;
